timemachine/cpp/src/kernels/k_periodic_torsion.cuh

```diff
@@ -38,28 +38,25 @@ void __global__ k_periodic_torsion(
     int k_idx = torsion_idxs[t_idx * 4 + 2];
     int l_idx = torsion_idxs[t_idx * 4 + 3];
 
-    RealType rij[D];
-    RealType rkj[D];
-    RealType rkl[D];
+    RealType r0[D];
+    RealType r1[D];
+    RealType r2[D];
 
-    RealType rkj_norm_square = 0;
+    RealType r1_norm_square = 0;
 
     // (todo) cap to three dims, while keeping stride at 4
     for (int d = 0; d < D; d++) {
-        RealType vij = coords[j_idx * D + d] - coords[i_idx * D + d];
-        RealType vkj = coords[j_idx * D + d] - coords[k_idx * D + d];
-        RealType vkl = coords[l_idx * D + d] - coords[k_idx * D + d];
-        rij[d] = vij;
-        rkj[d] = vkj;
-        rkl[d] = vkl;
-        rkj_norm_square += vkj * vkj;
+        r0[d] = coords[i_idx * D + d] - coords[j_idx * D + d];
```

Comment:
While here, might be good to use the same terminology for the reference and the kernel. Here you use i, j, k, l and in the reference you use xa, xb, xc, xd